migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc

```diff
@@ -1,16 +1,23 @@
 [id="planning-considerations-3-4"]
```

Comment:
I admit this is a bit of hassle, but for consistency with other redundant files such as `about-mtc.adoc` and `about-mtc-3-4.adoc`, it might be kinder to a maintenance writer if you rename this file to `network-considerations-mtc-3-4.adoc` and revise the ID attribute one line 1 and the context attribute on line 4.

Reply:
@mikemckiernan Vikram said we should not rename this assembly because it has been around for a while. He does not want to set up redirects.